packages/webservice/app/src/main/kotlin/com/yahoo/navi/ws/security/ExampleUserDetails.kt

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms
+ */
+
+package com.yahoo.navi.ws.security
+
+import com.yahoo.navi.ws.models.beans.User
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import org.springframework.security.core.userdetails.UserDetails
+
+class ExampleUserDetails(user: User) : UserDetails {
```

Comment:
Instead of ExampleUserDetails, NaviUserDetails

Reply:
updated